prime-router/metadata/HL7/catchall/hl7/codesystem/ExtensionUrlMapping.yml

```diff
@@ -297,9 +303,11 @@
   url: "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Component"
 
 # Date time as hl7v2 string
-# This extension exists because HL7 dates have very little restrictions (i.e. YYYY-MM, YYYY-MM-DD HH, YYYY-MM-DD HH:MM)
-# are all valid, but the FHIR specs has much stricter rules.  In order to reliably generate the same HL7 message, this extension
-# is used to capture what was in the original HL7 message and then exclusively used when mapping FHIR->HL7
+# These extensions exist because HL7 dates have very little restrictions (i.e. YYYY-MM, YYYY-MM-DD HH, YYYY-MM-DD HH:MM)
+# are all valid, but the FHIR specs has much stricter rules.  In order to reliably generate the same HL7 message, these
+# extensions are used to capture what was in the original HL7 message and then exclusively used when mapping FHIR->HL7
+- id: "hl7v2-date"
```

Comment:
This date only extension was added for two reasons: HL7 DT (date) and DTM (datetime) are distinct datatypes and should have separate handling, and storing it as a different extension makes it so transforms that should only be applied to datetimes (such as timezone conversion) would not apply. So far, only the IN1 mappings have been updated to use this extension.